syntax = "proto3";
package prototypes;

message Status {
  uint32 pid = 1;
  bool success = 2; // true = subscriber kept up
  uint32 retry_number =
      3; // which attempt (0‐based); 0..N−1, or 0xFFFFFFFF for “stop”
}
